### DISCUSSION TITLE
music lagging when hosting on rasperry pi

### DISCUSSION THREAD

**Discussion by Spastii** [jagrosh/MusicBot#1091] [Troubleshooting]
Has anyone else hosted the music bot on rasperry pi? I tried to find information on how to fix it but couldnt find any. 
Edit: removed rtarded parts.

**Comment by lorivaro** [comment]
Does your Pi have a wireless or ethernet connection? Avoid using WiFi.

**Reply by Spastii** [reply]
im using ethernet connection. But i might have fixed the problem atleast for now. I changed java from 8 to 11 when running the bot and changed server locations on discord closer to home manually.
